snippets/python/ceaser_cipher.md

```diff
@@ -0,0 +1,122 @@
+# Ceaser Cipher - Python3
```

Comment:
```suggestion
# Caesar Cipher
```

Reply:
Looks like the spelling needs to be updated everywhere :)

Reply:
Done, I think I've already fixed all. let me know if there's something more :)

Reply:
Looks like you misspelled for `Caesar` everywhere. It's Caesar (and not Ceaser) :)

Reply:
oh Je! you're actually right and I didn't notice it hahaha really sorry. it's done.